packages/block-editor/src/store/actions.js

```diff
@@ -197,12 +197,37 @@ export function multiSelect( start, end ) {
  *
  * @return {Object} Action object.
  */
+export function wipeSelectedBlock() {
+	return {
+		type: 'WIPE_SELECTED_BLOCK',
+	};
+}
+
+/**
+ * Returns an action object used in signalling that the block selection is cleared.
+ *
+ * @param {Object} type   Object specifying if the clearing is destructive or not, default
```

Comment:
This type param is no longer user, it should be removed.

Reply:
Oups! Solved.